api/graphql/schema.graphql

```diff
@@ -498,6 +498,12 @@ type PoolCandidate {
     @with(relation: "educationRequirementWorkExperiences")
 }
 
+type PoolCandidateWithSkillCount {
+  id: ID! @self(key: "id")
+  poolCandidate: PoolCandidate! @self
+  skillCount: Int @rename(attribute: "skill_count")
```

Comment:
Could you add a comment noting that skillCount will be null unless the query is scoped by skillsAdditive or skillsIntersectional?

Reply:
Great suggestion, added in [add note for skillCount field value](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6534/commits/e88e593256e5e6b4e9c8d63270282739358bfe35)